lib/response.js

```diff
@@ -64,7 +64,18 @@ var charsetRegExp = /;\s*charset\s*=/;
  */
 
 res.status = function status(code) {
-  this.statusCode = code;
+  if (code > 99 && code < 1000) {
+    if (typeof code === 'string') {
+      console.log('Response status code set to a value with type String. ' +
```

Comment:
The deprecation should be making use of the same deprecation system currently as use, namely leveraging the `depd` pacakge. This has many, many benefits over `console.log`, which is why it is in use.

Reply:
Oh woops, this was just a copy paste error! I pasted over a version I was testing where I replaced deprecate with a log.